svm-callback/Cargo.toml

```diff
@@ -0,0 +1,17 @@
+[package]
+name = "solana-svm-callback"
```

Comment:
Should we call it callbacks (plural)?

Reply:
The crate's already reserved. Also, the struct itself is named singular. I'll like to keep it as is.